src/zarr/core/config.py

```diff
@@ -78,31 +88,24 @@ def enable_gpu(self) -> ConfigSet:
                 "order": "C",
                 "write_empty_chunks": False,
                 "v2_default_compressor": {
-                    "numeric": {"id": "zstd", "level": 0, "checksum": False},
-                    "string": {"id": "zstd", "level": 0, "checksum": False},
-                    "bytes": {"id": "zstd", "level": 0, "checksum": False},
```

Comment:
Should these and the other removals below get added to a deprecations config -https://donfig.readthedocs.io/en/latest/configuration.html#deprecations?

Reply:
IMO a follow-up PR is best since the most helpful/important next step is enabling community testing via a pre-release. So maybe just a todo or issue to resolve this comment?

Reply:
I have [an issue](https://github.com/zarr-developers/zarr-python/issues/3104) tracking config-related changes in 3.1, I will paraphrase and link to your suggestion there